README.md

```diff
@@ -1,2 +1,261 @@
 # operator-certification-operator
-A Kubernetes operator to provision the OpenShift Operator Pipeline.
+A Kubernetes operator to provision resources for the operator certification pipeline.
+# Requirements
+The certification operator requires that you have the following tools installed, functional, and in your path.
+- [Install](https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli) oc, the OpenShift CLI tool (tested with version 4.7.13)
+- [Install](https://tekton.dev/docs/cli/) tkn, the Tekton CLI tool (tested with version 0.19.1)
+- [Install](https://git-scm.com/downloads) git, the Git CLI tool (tested with 2.32.0)
+- The certification pipeline expects you to have the source files of your Operator bundle
+
+# Pre - Installation
+### Install Openshift pipelines operator
+* Log into your cluster's OpenShift Console with cluster admin privileges
+* Use the left hand menu to navigate to *Operators*
+* In the *Operators* submenu click on *OperatorHub*
+* Use the Filter/Search box to filter on *OpenShift Pipelines*
+* Click the *Red Hat OpenShift Pipelines* tile
+* In the flyout menu to the right click the *Install* button near the top
+* On the next screen "Install Operator" scroll to the bottom of the page and click *Install*
+
+### Add a new namespace where the following secrets need to be applied
+`oc new-project oco`
+
+### Add the kubeconfig secret which will be used to deploy the operator under test and run the certification checks.
+```
+  oc create secret generic kubeconfig --from-file=kubeconfig=$KUBECONFIG
+  ```
+### Configuring steps for submitting the results
+- Add the github API token to the repo where the PR will be created
+```
+oc create secret generic github-api-token --from-literal GITHUB_TOKEN=<github token>
+```
+- Add RedHat Container API access key
+  
+  This API access key is specifically related to your unique partner account for Red Hat Connect portal. Instructions to obtain your API key can be found: [here](https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/operator-cert-workflow.md#step-b---get-api-key)
+```
+oc create secret generic pyxis-api-secret --from-literal pyxis_api_key=< API KEY >
+```
+# Installation Steps
+Since this operator isn't in OperatorHub, the process to get it into a cluster is manaul at this point. 
+Please follow the below steps to get the operator into your cluster. Follow the steps from [here](docs/INSTALLATION.md)
+# <a id="execute-pipeline"></a>Execute the Pipeline (Development Iterations)
+There are multiple ways to execute the Pipeline.  Below are several examples but parameters and workspaces can be removed or added per your requirements. 
+
```

Comment:
```suggestion
Clone the pipelines repository locally so you have the necessary `input` files to pass to `tkn` commands.
```shell
git clone https://github.com/redhat-openshift-ecosystem/operator-pipelines
cd operator-pipelines
```
```

Reply:
We need the above here. ty!

Reply:
done